packages/playground/website/src/lib/redux-store.ts

```diff
@@ -235,6 +256,16 @@ listSites().then(
 		)
 );
 
+export function useAppSelector<T>(
+	selector: (state: PlaygroundReduxState) => T
+): T {
+	return useSelector(selector);
+}
+
+export function useAppDispatch() {
```

Comment:
Nice. This is much better than having to get the store and use its dispatch. Thank you.